Vagrantfile

```diff
@@ -48,7 +50,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |c|
     end
 
     if $mount_users_dir
-      config.vm.synced_folder $home, $home, id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      config.vm.synced_folder $home, $home, type: "nfs"
```

Comment:
Interesting, I have not thought about "exposing" the whole /Users dir. 

The vagrant vm runs locally on your laptop. It is not reachable from the internet as far as I know. 

Can you please clarify more what do you mean by "expose" ? I am wondering is there some aspect of vagrant vm's that I do not know about.